cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/category-input-screen.js

```diff
@@ -0,0 +1,154 @@
+import { observer } from 'mobx-react';
+import { Link } from 'react-router-dom';
+import { useState, useCallback, useEffect } from 'react';
+import { useStore } from '../../stores';
+import FrequencyInputs from '../../components/frequency-inputs';
+
+function CategoryInputScreen() {
+  const { InputWizardStore, CashFlowStore } = useStore();
+
+  const [step, setStep] = useState('Expenses');
+  const [category, setCategory] = useState('Housing');
+  const [frequency, setFrequency] = useState('monthly');
+  const [startDate, setStartDate] = useState('');
+  const [amount, setAmount] = useState(0);
+
+  const addNewCFEvent = useCallback((event) => {
+    event.preventDefault();
+    console.log('made it to addNewCFEvent');
+    var userInputDetails = {
+      step: step,
+      category: category,
+      frequency: frequency,
+      startDate: startDate,
+      amount: amount,
+    };
+
+    CashFlowStore.addNewCashFlowEvent(userInputDetails);
```

Comment:
This can't work, because you have the cash flow store either deleted or commented out of root store right now. The props you're passing to this method are also incorrect for a cash flow event and it won't pass validation. I'd just remove this callback.